src/app/api/routes/events.py

```diff
@@ -8,26 +8,43 @@
 router = APIRouter()
 
 
-@router.post("/", response_model=EventOut, status_code=201)
+@router.post("/", response_model=EventOut, status_code=201, summary="Create a new event")
 async def create_event(payload: EventIn):
+    """Creates a new event based on the given information
+
+    Below, click on "Schema" for more detailed information about arguments
+    or "Example Value" to get a concrete idea of arguments
+    """
     return await crud.create_entry(events, payload)
 
 
-@router.get("/{event_id}/", response_model=EventOut)
+@router.get("/{event_id}/", response_model=EventOut, summary="Get information about a specific event")
 async def get_event(event_id: int = Path(..., gt=0)):
+    """
+    Based on a event_id, retrieves information about the given event
+    """
     return await crud.get_entry(events, event_id)
 
 
-@router.get("/", response_model=List[EventOut])
+@router.get("/", response_model=List[EventOut], summary="Get the list of all events")
 async def fetch_events():
+    """
+    Retrieves the list of all events with each related information
```

Comment:
"all event and their information" would be clearer perhaps?

Reply:
"Retrieves the list of all events and their information" ? 

(I will update other descriptions once we are ok to keep consistency)

Reply:
Sounds good!

Reply:
Since you commented on the other threads afterwards, perhaps you missed the notification over here @fe51 :sweat_smile:

Reply:
Nope ;)